bridge/bridge/deploy/sagemaker.py

```diff
@@ -160,9 +160,12 @@ def teardown(self, scoped_resource_prefix=None):
                 self.sagemaker_client.delete_model(ModelName=model_name)
 
         try:
-            Bucket = self.bucket_name
-            logger.info(f"Removing S3 Bucket {Bucket}")
-            response = self.s3_client.delete_bucket(Bucket=Bucket)
+            logger.info(f"Removing all items in S3 Bucket {self.bucket_name}")
+            bucket_resource = boto3.resource("s3").Bucket(self.bucket_name)
```

Comment:
I think you need to use `self.s3_client`, which has been configured with the specific session profile if specified by the user. Do you know if you use any clients other than the ones created in the constructor anywhere else?